compose/.env

```diff
@@ -0,0 +1 @@
+OPENINFOMAN_URL=http://localhost:8984
```

Comment:
We can probably drop the .env for the local deployment, since we have a default.